src/MarDyn.cpp

```diff
@@ -150,10 +153,13 @@ int main(int argc, char** argv) {
 		// Print to file
 		std::string logfileNamePrefix(options.get("logfile"));
 		std::cout << "Using logfile with prefix " << logfileNamePrefix << std::endl;
```

Comment:
We now can already use the logger for this message ;)

Also, I'd print the full log file name instead of just the prefix. Either the full names from all ranks or the pattern like "logfilename_RN.log" only once from the root rank. The latter would be less spammy.

Reply:
Good poit - will update the code.